examples/bert/bert_train.py

```diff
@@ -371,11 +400,76 @@ def decode_record(record):
     return example
 
 
+def get_checkpoint_callback():
+    if FLAGS.checkpoint_save_directory is not None:
+        if FLAGS.use_cloud_storage:
+            storage_client = storage.Client()
```

Comment:
Can we just use GFile as a consistent interface? E.g. https://www.tensorflow.org/api_docs/python/tf/io/gfile/rmtree

GFile says it should support `gs://` prefix for the entire API.